index.html

```diff
@@ -130,17 +131,19 @@
   h2="Contributions from a strong ecosystem"
   text="
       <p>
-        seL4, and its related technologies, are distributed across dozens of
-        repositories, with contributions from developers around the world.
+        seL4 and its related technologies are hosted by the seL4 foundation with
+        contributions from developers around the world.
```

Comment:
foundation > Foundation
and I suggest:
seL4 and its related technologies hosted by the seL4 Foundation receive contributions from developers around the world.
(there are related technologies not hosted by the Foundation)
Or we drop it completely:
seL4 and its related technologies receive contributions from developers around the world.

Reply:
The latter "seL4 and its related technologies receive contributions from developers around the world" is more succinct.
If possible, it would be good make this sentence more active, maybe by turning it around, e.g.
Developers from around the world contribute to seL4 and its related technologies.

Reply:
I also like the shorter version, but I would not turn it around, because that shifts the focus. `receive` is at least grammatically already active, so for this one I'd leave it (I generally agree that we should avoid passive voice).

The other question is if we really want to emphasise the "around the world" part, because that is a scary proposition for people in the high assurance space. Coupled with the processes below it might be fine. Just wanted to point it out for June to think about.